### DISCUSSION TITLE
Anonymous user is able to access all topics

### DISCUSSION THREAD

**Discussion by suddy417** [tchiotludo/akhq#1398] [Q&A] [answered]
Hello 

In my akhq cluster anonymous user is able to access all topics add it and delete it 

I have already added this under security 
default-group: no-roles

But still I have the same issue 
Could you please help

**Comment by tchiotludo** [comment, marked as answer]
You need to enable security and set a jwt token, see [here](https://akhq.io/docs/configuration/authentifications/jwt.html)

**Reply by suddy417** [reply]
Id it really necessary to use jwt tokens? Any other alternative is possible? I am not using any jwt tokens but I’m using some secrets(passwords)

**Reply by tchiotludo** [reply]
Jwt token is only the way that akhq use to store the logged user internally.

**Reply by suddy417** [reply]
Okay, thank you. I will try it out

**Reply by suddy417** [reply]
https://github.com/tchiotludo/akhq/issues/820 

I am facing similar issue. Isn’t the answer on that issue the solution?
